test/emqx_protocol_SUITE.erl

```diff
@@ -19,114 +20,181 @@
 -compile(export_all).
 -compile(nowarn_export_all).
 
--include("emqx.hrl").
+-import(lists, [nth/2]).
 
--include("emqx_mqtt.hrl").
+-import(emqx_frame_SUITE, [parse/1,parse/2]).
 
 -include_lib("eunit/include/eunit.hrl").
 
--import(emqx_serializer, [serialize/1]).
+-include_lib1("common_test/include/ct.hrl").
+
+-include("emqx_mqtt.hrl").
+
+-define(TOPICS, [<<"TopicA">>, <<"TopicA/B">>, <<"Topic/C">>, <<"TopicA/C">>,
+                 <<"/TopicA">>]).
+
+-define(CLIENT2, ?CONNECT_PACKET(#mqtt_packet_connect{
+                                    username  = <<"admin">>,
+                                    clean_start = false,
+                                    password  = <<"public">>})).
+
+receive_messages(Count) ->
+    receive_messages(Count, []).
+
+receive_messages(0, Msgs) ->
```

Comment:
Msgs is always returned as it is passed in. Why pass it in at all?

Reply:
It is draft.😂